python/src/FunctionalChaosValidation_doc.i.in

```diff
@@ -0,0 +1,201 @@
+%feature("docstring") OT::FunctionalChaosValidation
+"Validate a functional chaos metamodel.
+
+Available constructors:
+    FunctionalChaosValidation(*functionalChaosResult, cvMethod*)
+
+    FunctionalChaosValidation(*functionalChaosResult, cvMethod, kParameter*)
```

Comment:
these can be dropped when they are nested: here we just need to mention that the last parameter is optional in the Parameters section

Reply:
Ok. Fixed at 9c0f7ba6dacbaeec69233d91fa049c36fd0bf3fe